### DISCUSSION TITLE
more Ukrainian BIBLES proposal, quick translations switch feature suggestion

### DISCUSSION THREAD

**Discussion by seesmof** [tim-hub/obsidian-bible-reference#231] [Ideas]
GOD bless you brother. it would be good to add more Ukrainian BIBLES since bolls.life supports them. here is a suggestion on how to modify `src/data/BibleVersionCollection.ts`:
```ts
export const BibleVersionCollectionUkrainian = [
  {
    key: "utt",
    versionName: "Українська Біблія LXX УБТ Рафаїла Турконяка (2011) 77 книг",
    language: "Ukrainian",
    apiSource: BibleAPISourceCollection.bollsLife,
  },
  {
    key: "hom",
    versionName: "Біблія Івана Хоменка (1963) 74 книги",
    language: "Ukrainian",
    apiSource: BibleAPISourceCollection.bollsLife,
  },
  {
    key: "ubio",
    versionName: "Біблія Івана Огієнка (1962) 66 книг",
    language: "Ukrainian",
    apiSource: BibleAPISourceCollection.bollsLife,
  },
];
```
we added HOM and UBIO BIBLES here, those are good faithful BIBLE translations also. gotta make sure it works with the API well also, but those abbreviations is how they're called in the API as well i think.

also, it would be good to add a command pallet command to quickly change translation. i read the BIBLE in English, but my native language is Ukrainian and sometimes i need Ukrainian BIBLE references, so the ability to quickly toggle between BIBLE versions would be useful.

GOD bless you brother, may JESUS CHRIST our HOLY LORD GOD ALMIGHTY keep guiding you and helping you and providing for you and your family always ✝️💗

**Comment by Allexik** [comment]
Totally support and especially agree with the quick translation feature. Also would suggest adding an Ukrainian CUV translation, I guess the key is cuv23: 
"CUV23, БІБЛІЯ Сучасний переклад © УБТ (2020-2023)"
